src/harness/reference_models/geo/utils.py

```diff
@@ -254,7 +254,7 @@ def PolygonsAlmostEqual(poly_ref, poly, tol_perc=10):
     True if the two polygons are equal (within the tolerance), False otherwise.
   """
   if isinstance(poly_ref, dict):
-    poly_ref = GeoJsonToShapelyGeometry(poly1)
+    poly_ref = GeoJsonToShapelyGeometry(poly)
```

Comment:
Sorry, I made this small mistake, as I changed name at the end.
This should be: 
```
 poly_ref = GeoJsonToShapelyGeometry(poly_ref)
```
Basically here we are making sure that all inputs are shapely geometries, so that we can use the mathematical routines on them.

Reply:
No problem, sorry I should have told you about it before fixing it here.  I made the correct update.
I don't know if we need to add another PR only for correctign this on the master or we can wait for this test to be approved/merged before having this update on the master.

Reply:
Only your test code is using it, so if it gets pushed soon we can wait. Otherwise I will make a small update, as I also look at these winding thing.